app/views/static_pages/index.html

```diff
@@ -3,17 +3,16 @@
 
 
 <div id="tab-locations">
-  <p>this will be the locations tab.</p>
-
-  <div class="locations-img-central">
-    <button class="btn btn-default btn-lg" type="button">Central</button>
+  
+  <div class="col-xs-4 locations-img-central">
+    <img src="assets/illustrator/location_central.jpg" alt="Central">
```

Comment:
You should use the ruby image tag `<%= image_tag "image_path" %>`. You should also change the file extension to `index.html.erb`